packages/react-examples/src/react-experiments/UnifiedPeoplePicker/UnifiedPeoplePicker.Example.tsx

```diff
@@ -62,6 +63,48 @@ export const UnifiedPeoplePickerExample = (): JSX.Element => {
 
   const ref = React.useRef<any>();
 
+  const suggestionProps = useConst(() => {
+    return {
+      /*headerItemsProps: [
+        {
+          renderItem: () => {
+            return <>People Suggestions</>;
```

Comment:
please remove this if it is not needed

Reply:
So I was thinking leave them there so users can comment/uncomment to see the various options for headers and footers. I should probably add a comment explaining that though